docker/oso-monitor-sso/src/root/config.yml

```diff
@@ -1,10 +1,27 @@
 ---
 - hosts: localhost
-  gather_facts: no
+  # need this for ansible_nodename
+  gather_facts: yes
   vars_files:
   - /root/default_vars.yml
   vars:
 
   tasks:
-  - name: Placeholder task
-    debug: msg="This task does nothing"
+  - include_vars:
+      file: "{{ item.file }}"
+      name: "{{ item.name }}"
+    with_items:
+      - name: zagg_configdata
+        file: /secrets/zagg-config-values.yaml
```

Comment:
A secret sticks this file here. It just has the url, username, password, etc. So we make ansible read that in as vars so the data can be used to populate the config file template.